### ISSUE TITLE
noisy tar output on linux given mac built 2.40010 tarball

### ISSUE THREAD

**Issue by dz-at-tc** [net-amqp-rabbitmq/net-amqp-rabbitmq#236]
As you can see below, extracting the current tarball on Linux generates quite a bit of line noise, apparently not liking the extra attributes that Macs inject.

It still seems to work fine, but it would be nice not to have a big stream of warnings like that, mais non?

/DZ

```Mon Jun 27 13:45:20 EDT 2022
--> Working on Net::AMQP::RabbitMQ
Fetching https://cpan.metacpan.org/authors/id/M/MS/MSTEMLE/Net-AMQP-RabbitMQ-2.40010.tar.gz ... OK
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.lastuseddate#PS'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.lastuseddate#PS'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.lastuseddate#PS'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.lastuseddate#PS'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
/usr/bin/tar: Ignoring unknown extended header keyword `LIBARCHIVE.xattr.com.apple.metadata:_kMDItemUserTags'
Configuring Net-AMQP-RabbitMQ-2.40010 ... OK
==> Found dependencies: Math::Int64
--> Working on Math::Int64
Fetching https://cpan.metacpan.org/authors/id/S/SA/SALVA/Math-Int64-0.54.tar.gz ... OK
Configuring Math-Int64-0.54 ... OK
Building Math-Int64-0.54 ... OK
Successfully installed Math-Int64-0.54
Building Net-AMQP-RabbitMQ-2.40010 ... OK
Successfully installed Net-AMQP-RabbitMQ-2.40010
2 distributions installed

real	0m14.286s
user	0m8.249s
sys	0m1.685s
Mon Jun 27 13:45:34 EDT 2022
```

**Comment by manchicken** [comment]
That's wild; I haven't been able to see that and I can't see it on my local Linux box either.

**Comment by rawleyfowler** [comment]
@manchicken You can use `tar --no-xattrs ...` when tarring the project on BSD or Mac to avoid this

**Comment by manchicken** [comment]
This was fixed in 2.40011.
